RecoEgamma/EgammaPhotonProducers/python/conversionOpenTrackSequence_cff.py

```diff
@@ -0,0 +1,18 @@
+import FWCore.ParameterSet.Config as cms
+
+import RecoEgamma.EgammaPhotonProducers.conversionTrackProducer_cfi
+
+#producer from lowPt electronTracksOpen
+gsfTracksOpenConversionTrackProducer = RecoEgamma.EgammaPhotonProducers.conversionTrackProducer_cfi.conversionTrackProducer.clone(
```

Comment:
Why not simply
```
gsfTracksOpenConversionTrackProducer = conversionTrackProducer.clone(
```

Reply:
No, at ther end I agreed with Nancy that it should be better let as such (the same as in all other configs in this package)